### ISSUE TITLE
Ability to track current applications

### ISSUE THREAD

**Issue by egnwd** [docsocsf/sponsor-portal-OLD#18]
Ability to show applications you have open, based on the current idea for the job board this might be quite manual so will need some thought as to how we can make this easy/usable.

@jamesprinc3 had a few ideas about this

**Comment by levex** [comment]
Hi,

On Tue, Aug 15, 2017 at 03:42:37AM -0700, Elliot Greenwood wrote:
> Ability to show applications you have open, based on the current idea for the job board this might be quite manual so will need some thought as to how we can make this easy/usable.

I think this is an over-kill. For now we just want to have a simple
job-board like experience, no?

Best,

Lev

**Comment by egnwd** [comment]
@levex I agree, I created the issue as it's in the specification that was posted, but I've not added this issue to the Version 1.0.0 Milestone
